examples/springboot-avro-kafkakms/build.gradle

```diff
@@ -45,12 +45,15 @@ dependencies {
 
 
 compileKotlin {
+	dependsOn(":examples:springboot-avro-kafkakms:generateAvroJava")
```

Comment:
is this Avro related change really needed? 
is it a concicous change?

Reply:
It was not, removed!